README.md

```diff
@@ -67,11 +44,12 @@ sudo podman run \
     quay.io/centos-bootc/bootc-image-builder:latest \
     --type qcow2 \
     --config /config.json \
-    --local \
-    localhost/bootc:eln
+    quay.io/centos-bootc/fedora-bootc:eln
 ```
 
-When using the --local flag, we need to mount the storage path as a volume. With this enabled, it is assumed that the target container is in the container storage.
+Note: we need to mount the storage path as a volume, since using containers-storage is the new default. If the target container is not
+already in the local store, `bootc-image-builder` will attempt to copy it into containers-storage. The `--remote` flag can
+be used to skip this and use the image from the remote registry.
```

Comment:
Hmm I think what we should document first is noting the new behavior that sometimes (but definitely not always, one will want to e.g. `podman pull` the updated image first.

Let's just mention that, as one might think to use `--remote` for this, but I think that's more for a distinct build system oriented flow.

Reply:
We could also of course add a `--pull=` flag, just like `podman run` has.  For the bootc self-install flow, we get that for free because it is just exactly a `podman run` invocation of the container itself.